Resources/translations/AdminLTEBundle.ja.xliff

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file date="2017-01-12T20:00:00Z" source-language="en" target-language="ja" datatype="plaintext" original="none">
```

Comment:
Can you change the "date" and the original="AdminLTEBundle.en.xliff"

Reply:
done

Reply:
so the change needs to be in here in the .ja file